web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -438,9 +503,9 @@ export default function ProposalSettingsSection({
           isLoading={isLoading}
           onChange={changeBoot}
         />
-        <ProposalSpacePolicyField
-          policy={settings.spacePolicy}
-          actions={settings.spaceActions}
+        <SpacePolicyField
```

Comment:
I have been thinking on drop the `Proposal` prefix from these fields too. In fact, I talked about it with @ancorgs in a call discussing about the PBI I'm working on. However, I believe would be better to move it field to its own file. We can see it in the context of my PBI.